online_store/gt_sport_v2.js

```diff
@@ -0,0 +1,182 @@
+      <p> This ${sportProducts[i].name} is suitable for teenagers on ${sportProducts[i].level} level.</p>
+      <p> This product is currently out of stock / not available. </p>
+      `;
+    }
+    else if (sportProducts[i].availability === 1) {
+      descriptions.innerHTML = `
+      <p> <b>${sportProducts[i].name}</b></p>
+      <p> <img src='${sportProducts[i].imgSrc}' style="width:50px"><p>
+      <p> <b>Brand:</b> ${sportProducts[i].brand}</p>
+      <p> <b>Price:</b> ${sportProducts[i].price} Eur</p>
+      <p> Sold by a <b>gT partner</b> called ${sportProducts[i].seller}.</p>
+      <p> This ${sportProducts[i].name} is suitable for teenagers on ${sportProducts[i].level} level.</p>
+      <p> Last chance! Only 1 more left in stock! </p>
+      `;
+    }
+    else if (sportProducts[i].availability > 1 && sportProducts[i].availability <= 3) {
+      descriptions.innerHTML = `
+      <p> <b>${sportProducts[i].name}</b></p>
+      <p> <img src='${sportProducts[i].imgSrc}' style="width:50px"><p>
+      <p> <b>Brand:</b> ${sportProducts[i].brand}</p>
+      <p> <b>Price:</b> ${sportProducts[i].price} Eur</p>
+      <p> Sold by a <b>gT partner</b> called ${sportProducts[i].seller}.</p>
+      <p> This ${sportProducts[i].name} is suitable for teenagers on ${sportProducts[i].level} level.</p>
+      <p> We have only ${sportProducts[i].availability} pieces left. Hurry! </p>
+      `;
+    }
+    else {
+      descriptions.innerHTML = `
+      <p> <b>${sportProducts[i].name}</b></p>
+      <p> <img src='${sportProducts[i].imgSrc}' style="width:50px"><p>
+      <p> <b>Brand:</b> ${sportProducts[i].brand}</p>
+      <p> <b>Price:</b> ${sportProducts[i].price} Eur</p>
+      <p> Sold by a <b>gT partner</b> called ${sportProducts[i].seller}.</p>
+      <p> This ${sportProducts[i].name} is suitable for teenagers on ${sportProducts[i].level} level.</p>
+      <p> This product is currently available, ${sportProducts[i].availability} pieces in stock. </p>
+      `;
+    }
+  
+    const addToShoppingCartBtn = document.createElement('button');
+    addToShoppingCartBtn.textContent = 'Add to shopping cart';
+    addToShoppingCartBtn.addEventListener('click', function () {
+      shoppingCart.push(product);
+      console.log(shoppingCart);
+      addToShoppingCartBtn.setAttribute("disabled", "true");
+    });
+    descriptions.appendChild(addToShoppingCartBtn);
+    productDescription.appendChild(descriptions);
+    
+    if (sportProducts[i].availability ===0) {
+      addToShoppingCartBtn.setAttribute("disabled", "true")
+    }
+  }
+};
+
+productDescriptionDisplay();
+
+function addProductsToCart(product) {
+  shoppingCart.push(product);
+};
+
+function displayContentInCart() {
+  for (let i = 0; i < shoppingCart.length; i++) {
+    let cartList = document.createElement("div");
+    intoSCbtn.addEventListener("click", function () {
+      cartList.innerHTML = `
+      ${shoppingCart.name}:
```

Comment:
shoppingCart is an array and does not have a name property. Perhaps you meant to access an item of the array?

Reply:
I discussed it with Ivan, I had the same question, but because shoppingCart takes identically structured data from sportProducts, then apparently it should have .name and any other properties that sportProducts have...what do you think?

Reply:
resolved based on your suggestion, thanks.